UPGRADE.md

```diff
@@ -0,0 +1,114 @@
+# Upgrade to Ceramic One from Kubo
+
+This document describes how to upgrade an existing Ceramic node using Kubo to run with `ceramic-one` while preserving existing data.
```

Comment:
Can we be more precise here? It's really how to upgrade a js-ceramic node from using Kubo to using ceramic-one.